test/fixtures/attribute-generation-classname-rewriting.js

```diff
@@ -34,6 +36,9 @@ export default () => (
     <div {...props} {...rest} />
     <div {...props} data-foo {...rest} />
     <div {...props} className={'test'} data-foo {...rest} />
+    <Element styled-jsx />
```

Comment:
I won't want to include this feature here for now, let's fix one thing at the time